packages/mobile/src/components/oauth/OAuth.tsx

```diff
@@ -169,11 +169,25 @@ const TIKTOK_POLLER = `
 })();
 `
 
-const OAuth = () => {
+type OAuthProps = {
```

Comment:
This is the webview popup, I just added in props to control it via component rather than via redux